cmd/osv-scanner/main_test.go

```diff
@@ -290,6 +294,101 @@ func TestRun(t *testing.T) {
 				Scanned %%/fixtures/locks-many/composer.lock file and found 1 packages
 			`,
 		},
+		// output format: sarif
+		{
+			name:         "Empty sarif output",
+			args:         []string{"", "--format", "sarif", "./fixtures/locks-many/composer.lock"},
+			wantExitCode: 0,
+			wantStdout: `
+            {
```

Comment:
nit: all of these should be intended a few levels less

Reply:
Just changed the indentation from spaces to tabs like the rest of the code, see if this looks correct now?